sql/characters/characters.sql

```diff
@@ -12,8 +12,21 @@ INSERT INTO `creature_template` (`entry`, `modelid1`, `name`, `subname`, `minlev
 INSERT INTO `npc_text` (`ID`, `text0_0`, `text0_1`) VALUES 
 (55002, 'Here you can select a character template which will Gear, Gem, Spec, and finish glyphs for your toon instantly.\r\n\r\nSelect your spec:', 'Here you can select a character template which will Gear, Gem, Spec, and finish glyphs for your toon instantly.\r\n\r\nSelect your spec:');
 
+
+INSERT INTO `command` (`name`, `security`, `help`) VALUES 
+('template reload', 3, 'Syntax: .template reload\nType .template reload to reload Template NPC database changes');
+
+INSERT INTO `command` (`name`, `security`, `help`) VALUES 
+('template save a', 3, 'Syntax: .template save a <spec>\nDumps your current gear and makes it the alliance <spec>s new template.');
+
+INSERT INTO `command` (`name`, `security`, `help`) VALUES 
+('template save h', 3, 'Syntax: .template save h <spec>\nDumps your current gear and makes it the horde <spec>s new template.');
```

Comment:
hmm what about changing the commands names to "save horde" and "save alliance" ? because the core automatically will choose "a" for "alliance" anyway and "ho" for "horde" (and "hu" for "human")